app/src/components/MovieListContainer/MovieListContainer.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import MovieList from './../MovieList/MovieList';
+import response from './../../../data/response.json';
+
+export default class MovieListContainer extends React.Component {
+  static sortListBy(list, criteria) {
+    return _.reverse(_.sortBy(list, criteria));
+  }
+
+  constructor() {
+    super();
+    this.state = {
+      movies: [],
+    };
+  }
+
+  componentDidMount() {
+    // TODO: write an API request, wrap everything in a promise
+    this.getMovies();
```

Comment:
I did not get it. When Promise will be resolved, you have to setState with received movies. SO now you can do: `this.setState({movies: response}`;

Reply:
That's done for 2 reasons:

1. I'll need to get another list of movies once the search parameters are updated, so separate reusable method made sense to me. 
2. Eslint throws an error when it sees this.setState call in the .componentDidMount method.